boards/nucleo-wl55jc/Makefile.dep

```diff
@@ -1,4 +1,8 @@
 ifneq (,$(filter stdio_uart,$(USEMODULE)))
   FEATURES_REQUIRED += periph_lpuart
 endif
+ifneq (,$(filter netdev_default,$(USEMODULE)))
+  USEMODULE += sx126x_stm32wl
+  USEMODULE += sx126x_rf_switch
+endif
```

Comment:
The later should be pulled in also when only using the driver, e.g `tests/driver_sx126x`

```suggestion
ifneq (,$(filter netdev_default,$(USEMODULE)))
  USEMODULE += sx126x_stm32wl
endif
ifneq (,$(filter sx126x_stm32wl,$(USEMODULE)))
  USEMODULE += sx126x_rf_switch
endif
```